src/main/java/it/gov/pagopa/payhub/activities/service/JaxbTrasformerService.java

```diff
@@ -0,0 +1,61 @@
+package it.gov.pagopa.payhub.activities.service;
+
+import it.gov.pagopa.payhub.activities.exception.ActivitiesException;
+import jakarta.xml.bind.JAXBContext;
+import jakarta.xml.bind.JAXBElement;
+import jakarta.xml.bind.JAXBException;
+import jakarta.xml.bind.Unmarshaller;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+import org.xml.sax.SAXException;
+
+import javax.xml.XMLConstants;
+import javax.xml.transform.Source;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+
+/**
+ * A service for transforming XML files into Java objects using JAXB.
+ * <p>
+ * This service supports optional validation of the XML against an XSD schema.
+ * It is lazily loaded to optimize startup time and uses SLF4J for logging.
+ * </p>
+ */
+@Lazy
+@Slf4j
+@Service
+public class JaxbTrasformerService {
```

Comment:
The fact that we are using JAXB is an implementation detail.
The transformation from XML to Object has a well defined name, or unmarshalling
```suggestion
public class XMLUnmarshallerService {
```